### ISSUE TITLE
FFPEG bug?

### ISSUE THREAD

**Issue by ngsp94** [insert-coin/CloudyPanelPlugin#10]
FFMPEG seems to be unable to handle frames with odd-width. Streams a black screen or messes up the frames, with or without encoding. There is no problem with even-width frames.

**Comment by joelngwt** [comment]
I don't think I've ever seen this problem before. I think this issue is invalid.

**Comment by 9gix** [comment]
@ngsp94 do you still encounter the issues from your computer?

**Comment by ngsp94** [comment]
Yes, although the problem has changed. Now an odd width will trigger a breakpoint when 2nd player joins. I think it may have something to do with integer division when splitting the width. The right side ends up with 1 less pixel than the left, resulting in illegal array access...

**Comment by joelngwt** [comment]
Fixed in my forked repository, commit: b2fadec96a53f92a7a3f7134eed9e214c4fc6847
